tests/unit/actions/test_create.py

```diff
@@ -103,3 +108,42 @@ def test_run_create_bails_if_repository_does_not_match():
             remote_path=None,
         )
     )
+
+
+def test_create_borgmatic_manifest_creates_manifest_file():
```

Comment:
Since this function doesn't have any asserts (implicit or explict), I'm not sure the name of the function is accurate right now. As written, this function is more of a "does not raise" test case.

Reply:
(I'll fix this to save the PR roundtrip. Probably just add a `.once()`...)